doc/sphinx/changes.rst

```diff
@@ -267,6 +267,10 @@ Tactics
   fixes `#11504 <https://github.com/coq/coq/issues/11504>`_
   and `#14090 <https://github.com/coq/coq/issues/14090>`_,
   by Lasse Blaauwbroek and Hugo Herbelin).
+- **Deprecated:**
+  the undocumented `new auto` tactic
+  (`#14528 <https://github.com/coq/coq/pull/14528>`_,
+  by Pierre-Marie Pédrot).
```

Comment:
Please insert after the last **Deprecated** and before the first **Added** in this category.

Reply:
@Zimmi48: Concretely, how can this be done?

Reply:
I thought my request was crystal clear but since it isn't, I'll take care of pushing the change.

Reply:
Done.

Reply:
Ah, ok. I did not realize that @ppedrot had modified `changes.rst` directly.